homeassistant/scripts/db_migrator.py

```diff
@@ -53,9 +47,13 @@ def print_progress(iteration: int, total: int, prefix: str='', suffix: str='',
         print("\n")
 
 
-def run(args) -> int:
+def run(script_args: List) -> int:
```

Comment:
Be welcomed. C:

On Aug 24, 2016 5:22 PM, "Johann Kellerman" <notifications@github.com>
wrote:

> In homeassistant/scripts/db_migrator.py
> <https://github.com/home-assistant/home-assistant/pull/2781#discussion_r76150617>
> :
>
> > @@ -53,9 +47,13 @@ def print_progress(iteration: int, total: int, prefix: str='', suffix: str='',
> >          print("\n")
> >
> >
> > -def run(args) -> int:
> > +def run(script_args: List) -> int:
>
> Will keep in mind, thanks!
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/home-assistant/home-assistant/pull/2781/files/4aa35df8ece347c7d8163dc8c1c4b3967244fe6e#r76150617>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AASjGc0ETs6HZBU7YaJUfT2tbMTG_g6Yks5qjMQ7gaJpZM4Jhgvv>
> .
>